### PR TITLE
Grafana admin password

### LINKED ISSUE #42: Set the admin grafana password at deployment using some env variable

### PR DESCRIPTION
**Pull request by sipian**
Fixes #42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ export AUTH_FILE=<path to service-account.json>
 export GCS_BUCKET=prow
 export GITHUB_ORG=prometheus
 export GITHUB_REPO=prometheus
+export GRAFANA_ADMIN_PASSWORD=password
```
Comment (krasi-georgiev) [inline README.md]: can we have this as  ```export GRAFANA_ADMIN_PASSWORD=$(openssl rand -hex 20)```

File: components/prow/manifests/prow_internals_2.yaml [hunk 2]
```diff
@@ -308,10 +308,10 @@ spec:
...
         env:
-          name: PROMBENCH_PROJECT_ID
-          value: "{{ .PROJECT_ID }}"
-          name: PROMBENCH_INGRESS_IP
-          value: "{{ .INGRESS_IP }}"         
+          - name: PROMBENCH_PROJECT_ID
+            value: "{{ .PROJECT_ID }}"
+          - name: PROMBENCH_INGRESS_IP
+            value: "{{ .INGRESS_IP }}"
         args:
```
Comment (krasi-georgiev) [inline components/prow/manifests/prow_internals_2.yaml]: I will submit this in a seprate PR so can remove it from this one.

File: README.md (hunk 1 above)
Comment (sipian) [reply README.md]: If we set it to random value then the user deploying grafana will not know the admin password.
Will be unable to login to grafana-admin portal

Comment (krasi-georgiev) [reply README.md]: they can always do `echo $GRAFANA_ADMIN_PASSWORD`

Comment (krasi-georgiev) [reply README.md]: usually this will also be added to the Lastpass account.

**Pull request merged**